src/rust/rust-proto/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+pub(crate) mod graplinc {
```

Comment:
in general, could you add some comments on this library like "every time you add a new subsection to protos, you gotta update this too!" or something

Reply:
Yeah I don't fully understand what's going on here either. @colin-grapl mind weighing in?

Reply:
so if you have multiple proto modules that creates multiple rust modules, and it's expected that your module layout matches your proto module layout

so like, you have this module grapl.api.graph.v1beta1 that must be in a rust module structure that matches it otherwise, if you have protobufs that are imported by other protobuf modules it will fail to resolve

If you add any new modules they must go here and match their proto module path